W-11501790-SlackV11TriggerQ2-ESB (#297)
mulesoft/docs-composer: en/mulesoft/modules/ROOT/pages/ms_composer_ms_release_notes.adoc

New versions are automatically available once they are released; you don't need to perform any tasks to move to the new version.

== July 29, 2022

include::_partials/release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+3]

* The *On New Message* trigger is now available for Slack Connector.

//* You can now connect a new system: Salesforce Marketing Cloud Connector.
//+

//See xref:ms_composer_reference.adoc[the product documentation] for details about each connector.

include::_partials/release-notes/rn-headings.adoc[tag=knownIssues,leveloffset=+3]

==== Composer Platform

[%header%autowidth.spread]

|===

|Known Issue|Workaround |ID

> |When deleting a connection, Composer ensures that there are no flows using that connection. If there are flows using that connection, they appear in a list in the *Delete Connection* window.  However, in rare cases, flows created or edited before July 12, 2022 may not appear in the list. | |

Reviewer: ```suggestion
|When deleting a connection, Composer ensures that no flows use that connection. If flows use that connection, they appear in a list in the *Delete Connection* window.  However, in rare cases, flows created or edited before July 12, 2022 may not appear in the list. | |
```
